projects/personality_captions/interactive.py

```diff
@@ -275,8 +275,6 @@ def setup_interactive():
         )
     opt['task'] = 'parlai.agents.local_human.local_human:LocalHumanAgent'
     opt['image_mode'] = 'resnet152'
-    opt['no_cuda'] = True
```

Comment:
I'm happy with this but was it contributing to the bug?

Reply:
totally fine with the change, can't remember myself why it was in there in the first place